li-utils/src/main/java/com/linkedin/common/urn/DataProcessUrn.java

```diff
@@ -12,27 +12,27 @@
 public class DataProcessUrn extends Urn {
     public static final String ENTITY_TYPE = "dataProcess";
 
-    private final String dataJobNameEntity;
+    private final String dataProcessNameEntity;
```

Comment:
Since this is already part of `DataProcessUrn` class, consider dropping the redundant `dataProcess` prefix to make it more concise?

Reply:
Code Updated